package/src/components/ImageGallery/ImageGallery.tsx

```diff
@@ -308,10 +308,11 @@ export const ImageGallery = <
     const attachmentImages =
       cur.attachments?.filter(
         (attachment) =>
-          attachment.type === 'image' &&
-          !attachment.title_link &&
-          !attachment.og_scrape_url &&
-          getUrlOfImageAttachment(attachment),
+          (attachment.type === 'giphy' && attachment.thumb_url) ||
```

Comment:
@santhoshvai what would be the recommended check for this? Using giphy version data

Reply:
`attachment.giphy?.url` would be the right url check

Reply:
and the giphys that were sent before the backend change would not have the field `giphy`.. but rather the same as image attachments. The giphy field also provides dimensions upfront which could be useful for the gallery.

Reply:
@vishalnarkhede I didn't understand, what change do I need to do here?

Reply:
Instead of checking attachment.thumb_url, better to check `attachment.giphy?.url`

Reply:
attachment.giphy has no property as `url`. It has giphy versions in the property, should I pass it as a param and use it to get the `url`?

Reply:
yes, apologies.. it is `attachment.giphy?.[version]?.url`

Reply:
so the image gallery must know the giphy version being used as well

Reply:
Version should come from a parameter @santhoshvai?

Reply:
`giphyVersion` is present in the `messagesContext`.. and that version must be used everywhere reference: https://github.com/GetStream/stream-chat-js/pull/921

Reply:
Done